packages/asc-ui/src/components/MenuToggle/Toggle.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render, fireEvent } from '@testing-library/react'
+import Toggle from './Toggle'
+import { KeyboardKeys } from '../../types'
+
+describe('Toggle', () => {
```

Comment:
Can you rename Toggle to MenuToggle everywhere applicable? It's kind of confusing to me. Am I looking at a test foor Toggle or MenuToggle? Also, why is there a test for MenuToggle in the Toggle component PR?

Reply:
yes I will do that

Reply:
finished !